src/plugins/es_ui_shared/static/forms/docs/core/form_hook.mdx

```diff
@@ -0,0 +1,214 @@
+---
+id: formLibCoreFormHook
+slug: /form-lib/core/form-hook
+title: Form hook
+summary: The heart of the lib. It manages your fields so you don't have to
```

Comment:
Maybe a semicolon could work here since these sub headings don't have `.`'s in them afaict

Happy for this to be ignored!

```suggestion
summary: The heart of the lib; It manages your fields so you don't have to
```